src/site/content/en/blog/nfc/index.md

```diff
@@ -128,8 +129,9 @@ To scan NFC tags, first instantiate a new `NDEFReader` object. Calling `scan()`
 returns a promise. The [user may be prompted] if access was not previously
 granted. The promise will resolve if the following conditions are all met:
 
-- User has allowed the website to interact with NFC devices when they tap their
-  phone.
+- It must be called in response to a user gesture such as a touch gesture or
```

Comment:
```suggestion
- It was only called in response to a user gesture such as a touch gesture or
```

Reply:
This is to follow the best practice to express coordinate ideas in similar form. The specific change is to make all items in the list use the same verb tense.

Reply:
Thank you!